### PR TITLE
feat: APP-18 upload multiple photos in project media form gallery photo by selecting multiple files in file browser

### PR DESCRIPTION
**Pull request by blushi**
## Description

https://regennetwork.atlassian.net/browse/APP-18?atlOrigin=eyJpIjoiNzRhZmVhZjU0OTczNDMwYTk3OWM0Zjk1MDE1N2Q4ZDMiLCJwIjoiaiJ9

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

1. From https://deploy-preview-2346--regen-marketplace.netlify.app/ log in
2. Create or edit an existing project, go to the Media form
3. For the gallery photos, try selecting multiple files to upload (either drag and drop or select multiple)

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-components/src/components/inputs/new/FileDrop/FileDrop.tsx [hunk 1]
```diff
@@ -156,11 +157,12 @@ const FileDrop = forwardRef<HTMLInputElement, FileDropProps>(
       });
 
       if (result) {
-        setValue(result, currentFile.type, fieldIndex);
+        setValue(result, currentFile.type, fieldIndex + nextFieldIndex);
```
Comment (blushi) [inline web-components/src/components/inputs/new/FileDrop/FileDrop.tsx:160]: this was mostly already implemented but there was a small bug when setting the value that would replace the first uploaded photo with the next one

**Comment by blushi** [comment]
@erikalogie @clevinson see testing instructions

**Comment by erikalogie** [comment]
LGTM!

**Review by aaronc — APPROVED** [review]
utACK

**Pull request merged**